<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginPlugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>312</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>463</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>387</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="icon1">
        <property name="styleSheet">
         <string notr="true">left: 840px;
top: 211px;
width: 98px;
height: 88px;
background-color: rgba(229,229,229,1);
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="icon2">
        <property name="styleSheet">
         <string notr="true">left: 960px;
top: 211px;
width: 78px;
height: 84px;
background-color: rgba(229,229,229,1);
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="title">
        <property name="styleSheet">
         <string notr="true">left: 806px;
top: 320px;
width: 308px;
height: 39px;
line-height: 19px;
color: rgba(9,172,253,1);
font-size: 28px;
text-align: center;
font-family: AlibabaPuHui-bold;
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QLabel" name="ground1">
        <property name="styleSheet">
         <string notr="true">left: 735px;
top: 537px;
width: 308px;
height: 15px;
background-color: rgba(255,255,255,1);
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="Backround" name="br" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="0" colspan="3">
          <widget class="QPushButton" name="LoginBtn">
           <property name="styleSheet">
            <string notr="true">left: 765px;
top: 693px;
width: 387px;
height: 36px;
line-height: 20px;
border-radius: 3px;
background-color: rgba(3,180,245,1);
color: rgba(16,16,16,1);
font-size: 16px;
text-align: center;
font-weight: bold;
font-family: -regular;
color: white;</string>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QLineEdit" name="Password">
           <property name="styleSheet">
            <string notr="true">left: 765px;
top: 509px;
width: 387px;
height: 45px;
border-radius: 2px;
background-color: rgba(255,255,255,1);
border: 1px solid rgba(118,183,231,0.3);
background: transparent;
font-size: 30px;
color: white;</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>19</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,14,1,5">
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="imag">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255,255,255,1);
background: transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">left: 779px;
top: 431px;
width: 85px;
height: 29px;
line-height: 29px;
color: rgba(9,172,253,1);
font-size: 20px;
text-align: left;
font-family: SourceHanSansSC-medium;</string>
             </property>
             <property name="text">
              <string>用户登录</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="title3">
             <property name="styleSheet">
              <string notr="true">left: 1068px;
font: 75 11pt &quot;Sitka Subheading&quot;;
top: 435px;
width: 84px;
height: 20px;
line-height: 20px;
color: rgba(209,209,209,1);
font-size: 12px;
text-align: left;
font-weight: bold;
font-family: -medium;
background: transparent;</string>
             </property>
             <property name="text">
              <string>欢迎您的使用</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="lostPwd_btn">
           <property name="font">
            <font>
             <family>SourceHanSansSC-light</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">left: 1088px;
top: 653px;
width: 60px;
height: 15px;
line-height: 17px;
color: rgb(209,209,209);
font-size: 12px;
text-align: right;
font-family: SourceHanSansSC-light;
background: transparent;</string>
           </property>
           <property name="text">
            <string>忘记密码?</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="1">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>191</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="styleSheet">
            <string notr="true">left: 797px;
top: 653px;
width: 48px;
height: 15px;
line-height: 17px;
color: rgb(209,209,209);
font-size: 12px;
text-align: left;
font-family: SourceHanSansSC-light;</string>
           </property>
           <property name="text">
            <string>记住密码</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLineEdit" name="Account">
           <property name="maximumSize">
            <size>
             <width>387</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">left: 765px;
top: 509px;
width: 387px;
height: 45px;
border-radius: 2px;
background-color: rgba(255,255,255,1);
border: 1px solid rgba(118,183,231,0.3);
background: transparent;
font-size: 30px;
color: white;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QLabel" name="ground2">
        <property name="styleSheet">
         <string notr="true">left: 735px;
top: 537px;
width: 450px;
height: 450px;
background-color: rgba(255,255,255,1);
background: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>632</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,1">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>167</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="tips">
       <property name="font">
        <font>
         <family>AlibabaPuHui-light</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">left: 806px;
top: 1008px;
width: 308px;
height: 39px;
line-height: 20px;
color: rgba(255,255,255,1);
font-size: 14px;
text-align: center;
font-family: AlibabaPuHui-light;
color:white;</string>
       </property>
       <property name="text">
        <string>墨斗三维内核提供计算服务</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backround</class>
   <extends>QWidget</extends>
   <header>Backround.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
